src/components/BloombergToFDC3/common.ts

```diff
@@ -0,0 +1,8 @@
+export const LINK_PREFERENCES_PATH = ["finsemble", "custom", "bloomberg", "links"];
+
+
+export const getDisplayName = (link) => {
+  const tailsPart = link.target.args?.tails ? ` ${link.target.args.tails}` : "";
+  const panelPart = link.target.args?.panel ? ` on panel ${link.target.args.panel}` : "";
+  return `${link.displayName ? link.displayName : link.target.id+tailsPart}${panelPart}`;

```

Comment:
```suggestion
  return `${link.displayName ?? link.target.id+tailsPart}${panelPart}`;
```

ditto

Reply:
fair enough this time I think (but not changing it as this is tested and theres not much in it)

Reply:
I might be wrong but I believe these values are not nullish. They're empty strings, so I think they pass the check and end up returning an empty string.

Reply:
Yep that was it Gerard!

Reply:
Emptry string is falsey but not nullish
```
let test = "";
if (!test){ console.log("empty string is falsey");}
console.log(test ?? "emtpry string is nullish");
```

will only output 'emptry string is falsey'